spec/services/carrier_url_spec.rb

```diff
@@ -89,12 +89,44 @@
           status: 200,
           body: {
             'A1B2C3D4E5' => {
-              'melhorenvio_tracking' => 'ME123456789BR'
+              'tracking' => 'ME123456789BR'
             }
           }.to_json
         )
 
       expect(url).to eq('https://melhorrastreio.com.br/rastreio/ME123456789BR')
     end
+
+    it 'returns empty URL' do
```

Comment:
Deixar o objetivo mais claro

```suggestion
    it 'does not return URL without tracking code' do
```